crates/uv-scripts/src/lib.rs

```diff
@@ -161,6 +194,8 @@ pub enum Pep723Error {
     Utf8(#[from] std::str::Utf8Error),
     #[error(transparent)]
     Toml(#[from] toml::de::Error),
+    #[error("Invalid filename supplied")]
+    InvalidFilename,
```

Comment:
Lets include the script path in this error message.

Reply:
Done